### ISSUE TITLE
Not working with Neos 4.3 as expected

### ISSUE THREAD

**Issue by kdambekalns** [mocdk/MOC.NotFound#20]
I realized today, that since Neos 4.3 this package does not longer work (reliably?).

The native 404 handling interacts in some way, so that it was always the homepage being shown. I fixed it by removing `moc/notfound` and using this instead:

```
#
# Extend error matcher to render the document with uriPathSegment `notfound`
# for exceptions with 4xx status code
#
error {
    @context.notfoundDocument = ${q(site).children('[instanceof Acme.Com:ErrorPage]').get(0)}

    4xx {
        @position = 'start'
        condition = ${statusCode >= 400 && statusCode < 500 && notfoundDocument}
        renderer = Neos.Fusion:Renderer {
            @context.node = ${notfoundDocument}
            renderPath = '/root'
        }
    }
}
```

**Comment by ComiR** [comment]
Please have a look at #19. I think this would also solve it for Neos 4.3.

**Comment by kdambekalns** [comment]
@ComiR I'd rather say, mark this package as caompatible with Neos < 4.3 and use the [native 404 handling](https://github.com/neos/neos-development-collection/pull/2374) for > 4.2

**Comment by dlubitz** [comment]
Unfortunately the neos native handling does not work with ~~multisites~~ multilanguage sites yet (AFAIK). See https://neos-project.slack.com/archives/C050C8FEK/p1557236639260800

**Comment by kdambekalns** [comment]
Ah, you mean multi-language… Well, see the linked discussion. :) Or rather it's new home: https://discuss.neos.io/t/native-404-handling-with-content-dimensions-multi-language-sites/4739?u=kdambekalns

**Comment by dlubitz** [comment]
Thanks for this.

**Comment by dlubitz** [comment]
Is fixed with #18 for Neos 4.3.
See also #21 as possible solution for Neos 5.x.
